### ISSUE TITLE
[BUG]: Face detection does not work with: anti_spoofing = True

### ISSUE THREAD

**Issue by CatonTheRoRo** [serengil/deepface#1312]
### Before You Report a Bug, Please Confirm You Have Done The Following...

- [X] I have updated to the latest version of the packages.
- [X] I have searched for both [existing issues](https://github.com/serengil/deepface/issues) and [closed issues](https://github.com/serengil/deepface/issues?q=is%3Aissue+is%3Aclosed) and found none that matched my issue.

### DeepFace's version

0.0.93

### Python version

3.11.9

### Operating System

Archlinux

### Dependencies

absl-py==2.1.0
astunparse==1.6.3
beautifulsoup4==4.12.3
blinker==1.8.2
certifi==2024.7.4
charset-normalizer==3.3.2
click==8.1.7
deepface==0.0.93
filelock==3.15.4
fire==0.6.0
Flask==3.0.3
Flask-Cors==4.0.1
flatbuffers==24.3.25
gast==0.6.0
gdown==5.2.0
google-pasta==0.2.0
grpcio==1.65.5
gunicorn==23.0.0
h5py==3.11.0
idna==3.7
itsdangerous==2.2.0
Jinja2==3.1.4
keras==3.5.0
libclang==18.1.1
Markdown==3.7
markdown-it-py==3.0.0
MarkupSafe==2.1.5
mdurl==0.1.2
ml-dtypes==0.4.0
mtcnn==0.1.1
namex==0.0.8
numpy==1.26.4
opencv-python==4.10.0.84
opt-einsum==3.3.0
optree==0.12.1
packaging==24.1
pandas==2.2.2
pillow==10.4.0
protobuf==4.25.4
Pygments==2.18.0
PySocks==1.7.1
python-dateutil==2.9.0.post0
pytz==2024.1
requests==2.32.3
retina-face==0.0.17
rich==13.7.1
six==1.16.0
soupsieve==2.6
tensorboard==2.17.1
tensorboard-data-server==0.7.2
tensorflow @ file:///home/user/deepface/tensorflow-2.17.0/bazel-bin/tensorflow/tools/pip_package/wheel_house/tensorflow-2.17.0-cp311-cp311-linux_x86_64.whl#sha256=c590b2c2f35651c22df0ee27764ed1bdb12a987c7e02d951add8932066b62fba
tensorflow-io-gcs-filesystem==0.37.1
termcolor==2.4.0
tf_keras==2.17.0
tqdm==4.66.5
typing_extensions==4.12.2
tzdata==2024.1
urllib3==2.2.2
Werkzeug==3.0.3
wrapt==1.16.0
youtokentome==1.0.6

### Reproducible example

```Python
DeepFace.stream(
    db_path = "db",
    enable_face_analysis = False,
    anti_spoofing = True
)
```

### Expected Result

It should detect faces.

### What happened instead?

It does not detect any face.

**Comment by tanpham380** [comment]
I have meet one time this Error. Just conflict Libary in python, I Has Resolve by create new Env and install again deepface.
I think problem In TF-Keras install error @serengil

**Comment by serengil** [comment]
@tanpham380 thank you for the feedback

seems dependency issue, not a bug of deepface itself

**Comment by CatonTheRoRo** [comment]
[tanpham380](https://github.com/tanpham380) [serengil](https://github.com/serengil) There is no error message describing the error, and there is no instruction on how to install deepface in a certain way. So, what should we do? How should we install deepface?

**Comment by tanpham380** [comment]
@CatonTheRoRo 
Step 1 remove pip cache 
Step 2 install deepface with pip install deepface 
Step 3 have fun with deepface... 
Something I neef pip install TF-Keras for deepface
This error I just do that

**Comment by CatonTheRoRo** [comment]
[tanpham380](https://github.com/tanpham380) I have built TensorFlow from the source because my machine does not support AVX instructions, then I got an error telling me to install tf-keras, then I installed tf-keras with pip.
That was my installation process, which I repeated several times to ensure a clean installation in new Python environments with no success.

**Comment by tanpham380** [comment]
@CatonTheRoRo  I Have build TensorFlow with pip, not try with source So I cannot have something wrong with your problem... How about you can downgrade your TensorFlow??

**Comment by CatonTheRoRo** [comment]
[tanpham380](https://github.com/tanpham380) [serengil](https://github.com/serengil) I tried it again on another machine that supports AVX instructions with a very standard installation (pip install deepface, pip install tf-keras), but I faced the same problem.
